docs/cypress/e2e/links.cy.ts

```diff
@@ -0,0 +1,137 @@
+import { VALIDATED_LINKS } from '../data/validatedLinks';
+import { REQUEST_GET_LINKS } from '../data/requestGetLinks';
+import { BASE_URL } from '../data/constants';
+
+let allLinks: string[] = [];
+const numberOfLinks = 119;
+const requestedLinks: Set<string> = new Set();
+
+before(() => {
+  cy.task('readSitemapLinks').then((links: string[]) => {
+    allLinks = allLinks.concat(links);
+  });
+});
+
+describe('Local Sitemap', () => {
+  it('should have 119 links', () => {
+    expect(allLinks.length).to.eq(numberOfLinks);
+  });
+});
+
+for (let i = 0; i < numberOfLinks; i++) {
+  describe(`check page ${i}`, () => {
+    it(`all links on page ${i} should work`, () => {
+      const link = allLinks[i];
+      cy.task('log', `🧪[TESTING...] page ${BASE_URL}/${link}`);
+      cy.visit(link || '/');
+      cy.get('a').each(hrefWorks);
+      cy.get('button').each(hrefWorks);
```

Comment:
Do we have `<button>` elements with an `href`? I feel like those should be errors.

Reply:
Right, we don't have `href` on `<button>`. I somehow remember we had, but, anyway, I removed the assertion to save more memory.